dvc/commands/experiments/ls.py

```diff
@@ -1,9 +1,13 @@
 import argparse
 import logging
 
+from scmrepo.exceptions import SCMError
```

Comment:
This is unneeded

Also, for future reference in DVC we prefer using `dvc.scm.SCMError` and not `scmrepo.exceptions.SCMError` (so that we end up raising an exception that derives from `DVCException`)